HIP/hip-865.md

```diff
@@ -0,0 +1,104 @@
+---
+hip: 865
+title: Add EVM Support for transient storage and memory copy Cancun opcodes
+author: Danno Ferrin (@shemnon)
+working-group: Nana Essilfie-Conduah <@nana-ec>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2024-01-24
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/872
+---
+
+## Abstract
+
+Update the Hedera network to add support for new Opcodes found in the Cancun fork of
+Ethereum Mainnet, namely TSTORE, TLOAD, and MCOPY
+
+## Motivation
+
+Hedera's goal of EVM Equivalence also includes the requirement to stay
+up-to-date with the current state of Ethereum Mainnet. This HIP addresses new
+opcodes added to the EVM that do not address any features relating to Blobs, or
+the Consensus Layer, or changes to existing opcodes.
+
+## Rationale
+
+The opcodes discussed in this HIP only exist within the EVM and do not interact
+with any novel features of ethereum mainnet (such as blobs and CL beacon root).
+It is expected that future versions of solidity, Vyper, and other languages
```

Comment:
Should we mention that Solidity `v0.8.24` includes supports for the opcodes described in this HIP? https://soliditylang.org/blog/2024/01/26/solidity-0.8.24-release-announcement/

Reply:
Yea, we are trying to get ahead of what happened with PUSH0.  Updated verbiage.